Source/WebKit/UIProcess/gtk/AcceleratedBackingStoreDMABuf.cpp

```diff
@@ -87,6 +97,11 @@ OptionSet<DMABufRendererBufferMode> AcceleratedBackingStoreDMABuf::rendererBuffe
         if (forceSHM && strcmp(forceSHM, "0"))
             return;
 
+        // Don't claim to support hardware buffers if we don't have a device to import them.
```

Comment:
That seems unrelated, to this patch -- while also important to have.

Reply:
Yes, more or less, I had to do that to ensure tests run in swrast mode here when using WTR directly